### PR TITLE
feat: rebranding 2023

### PR DESCRIPTION
**Pull request by ClaraLpresta**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -6,6 +6,9 @@
     <title>PrestaShop Live Demo</title>
     <meta name="description" content="Looking for a powerful shopping cart software? Try the PrestaShop Demo now and visualize it on mobile, tablet and desktop." />
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```
Comment (Quetzacoalt91) [inline public/index.html:8]: Do we still need to load this CSS file?

File: src/screen.scss [hunk 2]
```diff
@@ -20,11 +20,14 @@
 @import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;700&display=swap');
```
Comment (Quetzacoalt91) [inline src/screen.scss:20]: Is this font still needed as well?

File: src/components/DemoHeader.vue [hunk 3]
```diff
@@ -2,30 +2,32 @@
...
-      <span>{{ $t("explore-back") }}</span>
-      <img :alt="$t('explore-back')" src="../../public/switch-BO-FO.png" />
-    </router-link>
-    <router-link to="front" class="btn btn-explore btn-visible-small btn-explore-front"
-      @click.native="updateIframeLink"
-    >
-      <span>{{ $t("explore-front") }}</span>
-      <img :alt="$t('explore-front')" src="../../public/switch-BO-FO.png" />
-    </router-link>
+    <div  id="buttons">
```
Comment (Quetzacoalt91) [inline src/components/DemoHeader.vue:14]: I see some spaces that might be changed. Can you run `npm run lint`?

File: public/index.html (hunk 1 above)
Comment (ClaraLpresta) [reply public/index.html]: Yes, it can be deleted, thanks !

File: src/screen.scss (hunk 2 above)
Comment (ClaraLpresta) [reply src/screen.scss]: The same, it's no longer in use.

**Review by Quetzacoalt91 — APPROVED** [review]

**Pull request merged**